dts/bindings/pinctrl/raspberrypi,pico-pinctrl.yaml

```diff
@@ -121,3 +121,22 @@ child-binding:
         description: |
           The slew rate of a pin. 0 corresponds to slow, and 1 corresponds to fast.
           The default value is 0 (slow), as this is the power on reset value.
+      oe-override:
```

Comment:
please, use vendor prefix for custom properties.

Reply:
Updated to use rp2-, which seems consistent with the prefix used for defines in the header file.